ScoutSuite/output/data/html/partials/aws/services.ec2.regions.id.snapshots.html

```diff
@@ -7,7 +7,8 @@ <h4 class="list-group-item-heading">{{name}}</h4>
     <div class="list-group-item">
         <h4 class="list-group-item-heading">Information</h4>
         <ul>
-            <li class="list-group-item-text">Id: {{id}}</li>
+            <li class="list-group-item-text">ID: {{id}}</li>
+            <li class="list-group-item-text">ARN: {{arn}}</li>
```

Comment:
@JJmako here's an example of adding the ARN to the "partial" that gets displayed in the report. All you need to do once the ARN is implemented in the parsing is add a line like this one to the partial so it shows up in the report.

There are a few partials (like the one for elasticache clusters - https://github.com/nccgroup/ScoutSuite/blob/master/ScoutSuite/output/data/html/partials/aws/services.elasticache.regions.id.vpcs.id.clusters.html) which uses an old generic template, for those you won't need to do anything.